From 1bd1eb1082617924e2f711301b33c9591c741e8a Mon Sep 17 00:00:00 2001
From: Tom Herbers <mail@tomherbers.de>
Date: Sat, 6 Sep 2025 11:58:59 +0200
Subject: [PATCH 1/2] modules: update openwrt

6c7919471e octeon: ubnt-*: Add device specific DTS pruning
78763b7b84 lantiq: Fix VPE kernel patch
2180b1d6de rockchip: Add Realtek RTL8821CU support for the Radxa ROCK Pi E v3
e52a2ca694 rockchip: Add support for Radxa ROCK 4SE
1ebda1b0fa rockchip: Add support for Radxa ROCK 4C+
65269fd3fc mac80211: Fix build of rt2800soc
c46cccde28 uqmi: update to latest HEAD
e32ee37fda uqmi: update to latest HEAD
b4b9288f2a mac80211: Update to 6.12.44
faf4f1e49e mediatek: add factory image for ipTIME AX3000SM
8b0eb9c91d kernel: Fix kernel regression in local-broadcast routes
c676281e7e kernel: bump 6.6 to 6.6.103
c77f2b442e rockchip: fix eMMC corruption on NanoPC-T6 with A3A444 chips
2a2f1573a7 rpcd: update to git openwrt-24.10 branch HEAD
9de2da13bf kernel: mtk_eth_soc: fix tx vlan tag for llc packets
af3a18db95 tools: firmware-utils: update to Git HEAD (2025-08-03)
d9179345ea tools: firmware-utils: update to Git HEAD (2025-07-24)
4576d311f8 realtek: fix stall after restart of otto timer
203c9f063b realtek: avoid interrupt storm on mass packet receive
3db4d8f23a ath79: add calibtation variant for Xiaomi AIoT AC2350
3180294897 ipq-wifi: add entry for Xiaomi AIoT AC2350
e6bb072af5 ipq-wifi: update to Git HEAD (2025-08-25)
d247904482 ipq-wifi: update to Git HEAD (2025-08-12)
8e5c83ca85 ipq-wifi: update to Git HEAD (2025-06-23)
c4ea5c0732 ipq-wifi: add BDF for IgniteNet SS-W2-AC2600
777d90a935 ipq-wifi: update to Git HEAD (2025-06-13)
bd7ab1e9ce ipq-wifi: update to Git HEAD (2025-06-11)
6ae235bcd0 armsr: remove 'console=tty1' from kernel command line
20271bf505 armsr: add patch (hack) to enable all "default" consoles
a65ca44cb7 kernel: bump 6.6 to 6.6.102
406489fd83 kernel: bump 6.6 to 6.6.101
03fe596699 mac80211: extend connection monitoring for MLO
64bbe71219 mt76: update to Git HEAD (2025-08-27)
5c37d1b9a7 udebug: update to Git HEAD (2025-08-24)
3e0bcf3af7 udebug: update to Git HEAD (2025-08-17)
c1a8e48f3b udebug: update to Git HEAD (2025-08-15)
76eb96c875 wifi-scripts: fix reload handling with random macaddr
---
 modules | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules b/modules
index 96a3d8182d..b09996fa64 100644
--- a/modules
+++ b/modules
@@ -2,7 +2,7 @@ GLUON_FEEDS='gluon packages routing'
 
 OPENWRT_REPO=https://github.com/openwrt/openwrt.git
 OPENWRT_BRANCH=openwrt-24.10
-OPENWRT_COMMIT=6bf869db0fe181b28ff71f3145715adc47de1a6b
+OPENWRT_COMMIT=6c7919471edbabf1bf5f25162fbf067bd10694f3
 
 PACKAGES_GLUON_REPO=https://github.com/freifunk-gluon/packages.git
 PACKAGES_GLUON_COMMIT=bae34a0f53b25dbd691c57c8ab2d0b7cfe811517

From 076c819aeca48536d623e5e4a34947b2bc94a3e5 Mon Sep 17 00:00:00 2001
From: Tom Herbers <mail@tomherbers.de>
Date: Sat, 6 Sep 2025 11:59:09 +0200
Subject: [PATCH 2/2] modules: update packages

c07c3f06a libtorrent-rasterbar: update boost dependencies
e62d5f7ad owut: update to 2025.09.03
db7e54807 yt-dlp: bump to 2025.08.22
ca56e5c2d adblock-fast: update to 1.1.4-r12
f4a0ee01e libradcli: fix build options
d84d73c16 sing-box: Revert "sing-box: update to 1.12.3"
---
 modules | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules b/modules
index b09996fa64..0e38231e1c 100644
--- a/modules
+++ b/modules
@@ -9,7 +9,7 @@ PACKAGES_GLUON_COMMIT=bae34a0f53b25dbd691c57c8ab2d0b7cfe811517
 
 PACKAGES_PACKAGES_REPO=https://github.com/openwrt/packages.git
 PACKAGES_PACKAGES_BRANCH=openwrt-24.10
-PACKAGES_PACKAGES_COMMIT=bb70450a573d01e069e0f8679c380f96e4466567
+PACKAGES_PACKAGES_COMMIT=c07c3f06a900d6650d9c6a74fecbaaa12e19a43c
 
 PACKAGES_ROUTING_REPO=https://github.com/openwrt/routing.git
 PACKAGES_ROUTING_BRANCH=openwrt-24.10
